Automattic/wp-calypso
# [e2e] Use active tests file as source for e2e a/b tests

Redo of #31741 where I messed up git 🤦‍♂️ 

Changes proposed in this Pull Request
This changes e2e tests from using default config for abtests to using active-tests.js
Testing instructions
Ensure tests pass in CircleCI
Run locally an make sure that after setABTestControlGroupsInLocalStorage is run, all tests and defaults from active-tests.js are set properly in local storage
Before:
![image](https://user-images.githubusercontent.com/31110506/55097137-dd478a80-5080-11e9-8de8-5f4aa589053a.png)


After:
![image](https://user-images.githubusercontent.com/31110506/55097112-d15bc880-5080-11e9-8b7b-985d0999d403.png)
